cmd/siklu/bank_management.c

```diff
@@ -54,6 +56,27 @@ struct software_bank_t* bank_management_get_current_bank(void) {
 		goto fail_free;
 	}
 	
+	config_boot_tries_left = siklu_fdt_getprop_string(fdt, "/", PROP_BOOT_TRIES_LEFT, NULL);
```

Comment:
I think that think code should not be a part of a getter but rather a specific function(s) that handles this logic

Reply:
Do you mean that we should alter current bank before getter?

Reply:
No. I mean that code that handles boot should not reside inside of `bank_management_get_current_bank`. It needs to be on its own function (or multiple functions)

Reply:
Extracted to bank_management_switch_current_bank()